app/src/main/java/ch/heigvd/pro/b04/android/Home/Home.java

```diff
@@ -64,4 +65,8 @@ public void scanQR(View view) {
             startActivity(marketIntent);
         }
     }
+
+    public void clearEmojiCode(View view) {
+
+    }
```

Comment:
I'm not sure if this method is supposed to remain to be a placeholder for future work, or some remains of previous iterations. Maybe clarify it in the PR or create a dedicated issue ?

Reply:
It doesn't belong here!